core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -59,14 +56,11 @@ private boolean isKnoxGatewayDefinedInServices(Set<ServiceComponent> serviceComp
             .anyMatch(serviceComponent -> KnoxRoles.KNOX_GATEWAY.equals(serviceComponent.getComponent()));
     }
 
-    public String generateLoadBalancerEndpoint(Stack stack, LoadBalancerType type) {
+    public String generateLoadBalancerEndpoint(Stack stack) {
         StringBuilder name = new StringBuilder()
             .append(stack.getName())
             .append('-')
             .append(ENDPOINT_SUFFIX);
-        if (LoadBalancerType.PUBLIC.equals(type)) {
```

Comment:
unit test would be nice here

Reply:
I've created a unit test class for this service, and added test cases for some of the other logic. I should have had a unit test for the previous version of the method. But now that it's just a string appender I'm not sure what would be tested here. The previous logic wasn't actually used anywhere, since we don't actually use public load balancers anywhere. (Public load balancers are being added in https://github.com/hortonworks/cloudbreak/pull/9675, although the enums have been renamed there.)